src/db.rs

```diff
@@ -179,6 +182,51 @@ pub async fn run_migrations(client: &DbClient) -> anyhow::Result<()> {
     Ok(())
 }
 
+pub async fn run_scheduled_jobs(db: &DbClient) -> anyhow::Result<()> {
+    let jobs = get_jobs_to_execute(&db).await.unwrap();
+    println!("jobs to execute: {:#?}", jobs);
+    tracing::trace!("jobs to execute: {:#?}", jobs);
+
+    for job in jobs.iter() {
+        update_job_executed_at(&db, &job.id).await?;
+
+        match handle_job(&job.name, &job.metadata).await {
+            Ok(_) => {
+                println!("job succesfully executed (id={})", job.id);
```

Comment:
We'll want to remove non-tracing printing. Is there a reason you added it? It should be possible to test all this out with RUST_LOG=triagebot=trace, for example.

Reply:
Good catch, need to remove that. Thanks!